frontend/src/v5/ui/controls/inputs/multiSelect/multiSelect.component.tsx

```diff
@@ -17,12 +17,16 @@
 
 import { SearchSelect } from '@controls/searchSelect/searchSelect.component';
 import { SelectProps } from '../select/select.component';
+import { FormControl, FormHelperText } from '@mui/material';
 
-export const MultiSelect = ({ defaultValue = [], ...props }: SelectProps) => (
-	<SearchSelect
-		defaultValue={defaultValue}
-		renderValue={(val) => (val as any[]).join(', ')}
-		{...props}
-		multiple
-	/>
+export const MultiSelect = ({ defaultValue = [], className, helperText, ...props }: SelectProps) => (
+	<FormControl required={false} disabled={props.disabled} error={props.error} className={className}>
```

Comment:
The `SearchSelect` is based on the `Select` component, which already has a `FormControl` and a `FormHelperText`. Is it necessary to add them here as well?